cli/api.py

```diff
@@ -112,8 +124,8 @@ def initiate_assay_upload(assay_name: str, xlsx_file: BinaryIO) -> UploadInfo:
         )
 
 
-def _update_assay_upload_status(job_id: int, etag: str, status: str):
-    """Update the status for an existing assay upload job"""
+def _update_upload_status(job_id: int, etag: str, status: str):
+    """Update the status for an existing upload job"""
     url = _url(f"/assay_uploads/{job_id}")
```

Comment:
shouldn't this url also change?

Reply:
Nope! Analysis uploads are (perhaps confusingly) stored in the `assay_uploads` table in the database, and this is the default Eve endpoint. Do you think it's important that we store assay and analysis upload jobs separately? Alternatively, should we rename the `assay_uploads` table to be more general?

Reply:
I don't have strong opinion here. WDYT

Reply:
I'm not sure there's a strong reason to do this in the short term, but this is the sort of detail I could see tripping up a new developer on the project. I'll add a note to the backlog.